### ISSUE TITLE
Outlook Google Calendar Sync deleted almost my entire calendar

### ISSUE THREAD

**Issue by ctr** [phw198/OutlookGoogleCalendarSync#431]
**OGCS Version**: v2.6.0.0
**Installed or Portable**: Installed

**Problem Description**  
I have been using Outlook Google Calendar Sync for 2-3 weeks successfully. 1 desktop PC and 1 laptop PC. Both Win 10 Pro x64 machines. Both with Outlook 2016. And an Android phone (Honor 9). This has been working fine.

Last week, without warning, the sync process removed all except about 4 appointments from my google calendar. I do not know why this happened. I am concerned it may happen again. It was very inconvenient!

I have sent you my OGcalsync.log file via Cloudwok.

**Comment by phw198** [comment]
Hi @ctr, and sorry to hear of this.

Could you explain a bit more about "1 desktop PC and 1 laptop PC"? Do they each have their own local Outlook installation and mailbox? Your logfile appears to show it's not connected to Exchange, so just a local repository of calendar items? And are both laptop and desktop syncing two-way to a common Google calendar?

To be honest, if this is how it's been set up and running for 3 weeks, I'm a little surprised you got away with it this long. :confused: Unless you only recently changed one/both of the machines to two-way? Nevertheless, these are the supported configurations:

![](https://github.com/phw198/OutlookGoogleCalendarSync/raw/master/docs/images/supported_sync_configs.png)

It sounds like you had the 6th one from the top. FWIW, I am hoping to support more complex calendar syncs, but it's not there yet.

Have you been able to recover the situation? I would suggest:
* Make one system your primary (eg PC) and turn off the other (eg laptop)
* Set OGCS to one-way sync from Outlook to Google
* Turn on `Disable deletions` - at least temporarily
* Undelete the items from your deleted items in Outlook
* Sync them back to populate Google.
* Turn on two way sync and check it all behaves

If you want the laptop turned back on, you mustn't two-way sync both of them to the same Google calendar. 

HTH

**Comment by ctr** [comment]
Dear Paul,

Thanks for the explanation. I got my deleted items on my PC and will try 
to sync them soon.

I didn't at all understand that diagram of supported configurations when 
setting up. Perhaps it would be clearer if you added some text in the 
image above the rows showing the configurations saying:

"Supported?"

"PC1"

"Google"

"PC2,3,4..."


(I used to use Funambol for many years, which worked fine like this, but 
unfortunately the open-source client stopped working in Outlook 2013 
onwards and I can no longer use that.)

All the best,

Chris.
